### PR TITLE
Update iOS test sheet

### LINKED ISSUE #59: update iOS test run - include opening a link in Brave from twitter share picker
Add this scenario to test run:

Long press on a link in the Twitter app to get the share picker. Choose Brave. Verify link opens in Brave and Brave does not crash.

### LINKED ISSUE #69: Update iOS tests for favourites

### PR DESCRIPTION
**Pull request by srirambv**
Fixes #59 
Fixes #69

### CODE REVIEW

File: wikitemplate-ios.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+- [ ] Check the Brave version in About and make sure it is EXACTLY as expected
 
 ## Data
 
-- [ ] Make sure that data from the last version appears in the new version OK.
-- [ ] Test that the previous version's cookies are preserved in the next version.
+- [ ] Make sure that data from the last version appears in the new version OK
+- [ ] Test that the previous version's cookies are preserved in the next version
+- [ ] Test that saved passwords are retained upon upgrade
+- [ ] Ensure stats are retained when upgrading from previuos version
```
Comment (kjozwiak) [inline wikitemplate-ios.md]: nit: `previuos` should be `previous`

Comment (kjozwiak) [inline wikitemplate-ios.md]: nit: `previuos` should be `previous`

File: wikitemplate-ios.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 
 ## Installer
 
-- [ ] Check that installer is close to the size of last release.
-- [ ] Check the Brave version in About and make sure it is EXACTLY as expected.
+- [ ] Check that installer is close to the size of last release
+- [ ] Check the Brave version in About and make sure it is EXACTLY as expected
 
 ## Data
```
Comment (kjozwiak) [inline wikitemplate-ios.md]: Probably should add "Shield" settings retention per site in there as well.

**Comment by kjozwiak** [comment]
@srirambv everything looks good other than the small nit. We should probably add a case for testing shield setting retention per site as well.

**Comment by srirambv** [comment]
Updated the test to add shield settings in data section. Is that sufficient?

File: wikitemplate-ios.md [hunk 3]
```diff
@@ -9,7 +9,8 @@
 - [ ] Make sure that data from the last version appears in the new version OK
 - [ ] Test that the previous version's cookies are preserved in the next version
 - [ ] Test that saved passwords are retained upon upgrade
-- [ ] Ensure stats are retained when upgrading from previuos version
+- [ ] Ensure stats are retained when upgrading from previous version
+- [ ] Ensure per site settings are retained when upgrading from preivous version
```
Comment (LaurenWags) [inline wikitemplate-ios.md]: nit: change preivous to previous, otherwise LGTM 😄

**Comment by kjozwiak** [comment]
@srirambv LGTM other then the `preivous` --> ` previous` nit :)

**Comment by srirambv** [comment]
@kjozwiak @LaurenWags fixed the nits and updated the history test.  A quick scan and its ready to be merged

File: wikitemplate-ios.md [hunk 4]
```diff
@@ -21,12 +24,18 @@
 - [ ] Test that you are able to delete a bookmark folder with bookmarks inside
 - [ ] Test adding a bookmark domain subpaths is retained and you are successfully able to visit the domain subpath in a new tab
 
+## Favouritesf
```
Comment (kjozwiak) [inline wikitemplate-ios.md]: nit: change `Favouritesf` to `Favourites`

**Comment by srirambv** [comment]
Nailed the final nit. 😁

**Pull request merged**